src/pages/ProductList/ProductWrap.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
```

Comment:
```suggestion
import "./ProductWrap.scss"
```

컴포넌트를 분리하면 scss 파일도 함께 분리해서 관리해주세요!